x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.ts

```diff
@@ -146,6 +146,10 @@ export const useBulkAddToCaseActions = ({
             onClick: (alerts?: TimelineItem[]) => {
               selectCaseModal.open({
                 getAttachments: ({ theCase }) => {
+                  if (theCase == null) {
+                    return alerts ? casesService?.helpers.groupAlertsByRule(alerts) ?? [] : [];
```

Comment:
can `alerts` actually be missing? If this is triggered via the bulk actions it shouldn't ever, right?